### ISSUE TITLE
Accessibility - Parent ticket for Dashboard Pages

### ISSUE THREAD

**Issue by mnischay** [awslabs/iot-application#1202]
### Description
This is a parent story for all the accessibility stories/work. Please create sub tasks as and when required

### Acceptance Criteria
1. All the components in iot-application is WCAG AA compliant (https://www.digitala11y.com/wcag-checklist/)
2. Create a verification/Test plan
3. Tests at the least should include some e2e tests and should run on each PR similar to our existing e2e test cases
4. Future PR  builds which do not meet the criteria should fail

### Additional context
1. Please use any tool / plugins you need, I do not have a strong preference
2. We will look from you to lead this effort in terms of creating stories and corresponding implementation

**Comment by Pushpajyothi** [comment]
Accessibility issues observed in IoT Application :
 
- https://github.com/awslabs/iot-application/issues/1594
- https://github.com/awslabs/iot-application/issues/1595
- https://github.com/awslabs/iot-application/issues/1596
- https://github.com/awslabs/iot-application/issues/1597
- https://github.com/awslabs/iot-application/issues/1598
